### PR TITLE
Add and edit companies found in Australia and the world

### PR DESCRIPTION
**Pull request by Scottie0001**
Add new companies found in Australia and the world, edit Cotton On to edit name in OSM.

### CODE REVIEW

File: data/brands/shop/bag.json [hunk 1]
```diff
@@ -132,6 +132,17 @@
         "shop": "bag"
       }
     },
+    {
+      "displayName": "Strandbags",
+      "id": "",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "Strandbags",
+        "brand:wikidata": "Q1203Q111946652426",
```
Comment (Cj-Malone) [inline data/brands/shop/bag.json:141]: Presumably this is an error

Comment (AntiCompositeNumber) [reply data/brands/shop/bag.json:141]:
```suggestion
        "brand:wikidata": "Q111946652",
```
appears to be the correct one

Comment (Scottie0001) [reply data/brands/shop/bag.json]: That is correct, wikidata is Q111946652

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]
In general, `brand=`, `name=`, and the NSI `displayName` should all match unless there's a good reason for them not to. All should use OSM naming conventions, which may not match what is used on Wikidata or Wikipedia. Be careful that the locationSet value(s) you choose are correct.

In the future, smaller, more focused pull requests are easier to review.

File: data/brands/shop/clothes.json [hunk 2]
```diff
@@ -2293,6 +2315,7 @@
       "displayName": "Cotton On",
       "id": "cottonon-3937bd",
       "locationSet": {"include": ["001"]},
+      "preserveTags": ["^name"],
```
[the remark this answers is not in the document]
Comment (Scottie0001) [reply data/brands/shop/clothes.json:2318]: There are multiple brands of stores that Cotton On have that do not have a wikidata

File: data/brands/shop/bed.json [hunk 3]
```diff
@@ -306,6 +306,18 @@
...
       }
     },
+    {
+      "displayName": "Original Mattress Factory",
+      "id": "",
+      "locationSet": {"include": ["au"]},
+      "tags": {
+        "brand": "The Original Mattress Factory",
+        "brand:short": "OMF",
+        "brand:wikidata": "Q123014288",
```
[the remark this answers is not in the document]
Comment (Scottie0001) [reply data/brands/shop/bed.json:316]: Edited to reflect US stores, no wikidata tag for Australian company

File: data/brands/office/insurance.json [hunk 4]
```diff
@@ -1500,6 +1512,18 @@
       }
     },
     {
+      "displayName": "RACQ",
```
[the remark this answers is not in the document]
Comment (Scottie0001) [reply data/brands/office/insurance.json:1515]: Names to reflex more general traffic

**Comment by Scottie0001** [comment]
Changes have been made to reflect suggestions

File: data/brands/office/insurance.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (AntiCompositeNumber) [reply data/brands/office/insurance.json:1515]: I'm not sure what you mean.

[the remark this answers is not in the document]
Comment (Scottie0001) [reply data/brands/office/insurance.json:1515]: I have updated the name/brand to reflect the business name and suggestions. Previous comment was a typo mistake.

**Review by AntiCompositeNumber — APPROVED** [review]

**Pull request merged**